### ISSUE TITLE
UnboundLocalError -> disable [TRAIN][SAVE_IMG] -> ZeroDivisionErrpr

### ISSUE THREAD

**Issue by hongchanphat** [mimbres/neural-audio-fp#52]
I try to reproduce this repo with steps:

1. Setup env by  `conda env create -f environment.yml`
2. Download and unzip dataset by `kaggle datasets download -d mimbres/neural-audio-fingerprint `
3. Run training process by `python run.py train MY_CHECKPOINT`

then got error UnboundLocalError , try again with changing config from #43  got another error ZeroDevisionError

I tried with 2 machine with different cuda version but got same errors
- Ubuntu 20.04, NVIDIA Driver 565.77, GPU RTX A5000, CUDA 11.6, CUDDN8 8.4.0.27, Tensorflow 2.4.1, Python 3.8.10
- Ubuntu 20.04, NVIDIA Driver 535.230.02, GPU RTX 3090, CUDA 12.2, CUDDN 8.9.5.30, Tensorflow 2.4.1, Python 3.8.10

![Image](https://github.com/user-attachments/assets/cffb34e7-4b38-4b0b-97ea-42b8ac3fd814)

Could you share configuration on your machine (GPU, NVIDIA Driver, CUDA ...) let me reproduce if error happen? @mimbres

**Comment by mimbres** [comment]
@hongchanphat If the loss is 0 and you skipped training to go straight to validation, it means there’s no data. Check the directory after downloading the data.  Read the "Dataset installation" in the README.md...

**Comment by hongchanphat** [comment]
@mimbres I found rootcause: I run command `python run.py train CHECKPOINT_NAME` at root dir of project, but config file is relate path like "../neural-audio-fp-dataset/...", I changed to absolute path, it worked.

Thanks your suggestion, it help me find root cause ...
